abbyy_to_epub3/create_epub.py

```diff
@@ -1123,12 +1135,22 @@ def craft_epub(self, epub_outfile="out.epub", tmpdir=None):
             epub_outfile = '%s.epub' % epub_outfile
         epub.write_epub(epub_outfile, self.book, {})
 
-        # run checks
-        verifier = EpubVerify(self.debug)
-        if self.args:
-            if self.args.epubcheck:
-                self.logger.info("Running EpubCheck on {}".format(epub_outfile))
-                verifier.run_epubcheck(epub_outfile)
-            if self.args.ace:
-                self.logger.info("Running DAISY Ace on {}".format(epub_outfile))
-                verifier.run_ace(epub_outfile)
+        # run validation on epub
+        if self.args and self.args.epubcheck:
+            self.validate_epub(epub_outfile)
+
+    def validate_epub(self, epub_file):
+        self.logger.debug("Running EpubCheck on {}".format(epub_file))
+        result = self.verifier.run_epubcheck(epub_file)
```

Comment:
QUESTION: Is `result` even used?

Reply:
yes -- `if not self.debug` uses `result.messages`